standards-in-genomic-sciences.csl

```diff
@@ -182,7 +182,7 @@
           <text macro="editor" prefix=" " suffix="."/>
           <group prefix=" " suffix=".">
             <text variable="container-title" font-style="italic" form="short" suffix=" "/>
-            <group delimiter=";" prefix=" ">
+            <group delimiter=";">
               <date variable="issued">
                 <date-part name="year"/>
               </date>
```

Comment:
> But this can become more tricky if the only child node has children itself.

Does it though? I can't think of a case where that'd make a difference. Obviously if the group has affixes, those need to be passed down to the child.

> Moreover, AFAIK one can use group also for a conditional existence statement.

But only in a meaningful way if there are more than one child items, right? The conditional is that if you have a term and a variable in a group, the term only gets printed when the variable does. Or what did you have in mind?